queue/src/fiskaltrust.Middleware.Localization.QueueDE/JournalProcessorDE.cs

```diff
@@ -80,6 +80,7 @@ public async IAsyncEnumerable<JournalResponse> ProcessAsync(JournalRequest reque
             _logger.LogDebug($"Processing JournalRequest for DE (Type: {request.ftJournalType:X}");
             if (request.ftJournalType == (long) JournalTypes.TarExportFromTSE)
             {
+                request.MaxChunkSize = _middlewareConfiguration.TarFileChunkSize;
```

Comment:
Maybe we should only "overwrite" this in case it's not explicitely set in the request? I.e. only if it's the default value.🤔